src/Pulp/Files.purs

```diff
@@ -54,9 +60,23 @@ testGlobs :: Options -> AffN (Set String)
 testGlobs = globsFromOption "testPath"
 
 dependencyGlobs :: Options -> AffN (Set String)
-dependencyGlobs =
-  globsFromOption' (\path -> Path.concat [path, "purescript-*", "src"])
-                   "dependencyPath"
+dependencyGlobs opts = do
+  Project pro <- getOption' "_project" opts
+  -- If project file has a `set` property we assume it's a psc-package project file
+  case runExcept (readProp "set" pro.projectFile >>= readString) of
+    Right _ -> pscPackageGlobs
+    _ -> globsFromOption' (\path -> Path.concat [path, "purescript-*", "src"]) "dependencyPath" opts
+
+pscPackageGlobs :: AffN (Set String)
+pscPackageGlobs =
+  execQuiet "psc-package" ["sources"] Nothing <#> processGlobs
+  where
+    -- Split on newlines and strip the /**/*/.purs suffixes just to
+    -- append them later so it plays well with the other globs
+    processGlobs =
+      (split (Pattern "\r\n") >=> split (Pattern "\n")) >>>
+      mapMaybe (stripSuffix (Pattern "/**/*.purs")) >>>
```

Comment:
Ah yes thank you, I have updated this and the tests are green for me on Windows now, but unfortunately appveyor is failing because of github api rate limiting O_o